### PR TITLE
Patch 7.2.4

### PR DESCRIPTION
**Pull request by UTDZac**
I forgot to make separate PRs to review a few of these new features, hopefully these are small enough additions.

I'm once again skipping ahead to version 7.2.4 to help alert users of 7.0.3 that a new version is available, since 4>3.

**Whats new:**
- [Bizhawk] New [Time Machine](https://user-images.githubusercontent.com/4258818/210423115-a5582c6a-faeb-4da1-a2a6-ff1dbf962bef.png) feature!
   - This allows you to choose a point in time in the past to return to. 
   - This is helpful if you've ever loaded a savestate back to the beginning of the game on accident.
- During doubles battles, when targeting a Pokémon with a move, the Tracker now reflects move effectiveness based on which Pokémon is being targeted
- Added in Maxie and Archie boss battles to Log Viewer
- Added an option to remove the top Pokémon icon shown on the new game Tracker screen

Time Machine feature:
image

### CODE REVIEW

**Review by Fellshadow — APPROVED** [review]
Seems like a great feature, just gotta keep an eye out for if 10 is perhaps too many to keep in memory for some people. Idk if it *would* cause issues but you never know (if there are issues could potentially make an option to let the user enable/disable this feature and/or choose how many restore points to make?)

**Comment by UTDZac** [comment]
Good notes. I went ahead and made this an option that defaults to being enabled. I also moved the debug function to `Debug` and cleared up notes about usage.

File: ironmon_tracker/Utils.lua [hunk 1]
```diff
@@ -55,6 +55,35 @@ function Utils.reverseEndian32(value)
...
+	end
+
+	if changedAddrMessage ~= "" and (initialPrint or #Utils.prevAddrs > 0) and Utils.prevAddrsMessage ~= changedAddrMessage then
+		Utils.prevAddrsMessage = changedAddrMessage
+		changedAddrMessage = changedAddrMessage .. "----- ----- ----- -----"
+		print(changedAddrMessage)
+	end
+
+	Utils.prevAddrs = addrValues -- replace previous addresses and values with the new ones
+end
```
[the remark this answers is not in the document]
Comment (UTDZac) [reply ironmon_tracker/Utils.lua]: You're right. Cleared this up and moved it to `Debug.lua`

**Review by Fellshadow — APPROVED** [review]
changes look good to me

**Pull request merged**